includes/attendee/attendee-repository.php

```diff
@@ -128,4 +128,46 @@ function ( object $row ) {
 			$rows
 		);
 	}
+
+	/**
+	 * Add a user as host of an event.
+	 *
+	 * @param Attendee $attendee The attendee to add as host.
+	 * @return void
+	 */
+	public function add_host( Attendee $attendee ): void {
```

Comment:
Instead of having this function, I think we could use the existing `insert_attendee()` (it already handles the `is_host` flag). Personally I see the host as an attendee who has a special status, instead of the host being a separate entity.

The calling code is then responsible for deciding whether the attendee being created is a host or not, something like:

```php
$attendee = new Attendee( 1, 2 );
$attendee->mark_as_host();
$attendee_repository->insert_attendee( $attendee );
```

Reply:
The user is attending to the event, but it is not marked as host. If I do this, we will have two rows with the same event_id and user_id.